src/platform-integration/web/wasm.md

```diff
@@ -138,34 +163,56 @@ If the application doesn't load:
 1. Check the developer console for errors.
 1. Validate a successful build with the typical JavaScript output.
 
-[ff-preview]: https://www.mozilla.org/en-US/firefox/channel/desktop/
-
 ### Known limitations
 
 Wasm support has some limitations.
 The following list covers some common issues.
 
-#### Chrome 119 and Firefox 120 or later
-
+#### Chrome 119 or later
+ 
 As mentioned in [Load it in a browser](#load-it-in-a-browser), 
 to run Flutter web apps compiled to Wasm, 
-use _Chrome 119 or later_ or _Firefox 120 or later_.
+use _Chrome 119 or later_.
 
 Some earlier versions supported WasmGC with specific flags enabled,
 but WasmGC encodings changed once the feature was stabilized.
 To ensure compatibility, run the latest version of the Flutter `master` channel
-and the latest version of Chrome or Firefox.
+and the latest version of Chrome.
+
+- **Why not Firefox?**
+  Firefox versions 120 and later were previously able to run Flutter/Wasm, but
+  they're [currently experiencing a bug][] that is blocking compatibility with Wasm.
+- **Why not Safari?**
+  Safari does not support Wasm-GC yet; [this bug][] tracks their
+  implementation efforts.
+  
+{{site.alert.warning}}
+  Flutter with Wasm can't run on the iOS version of any browser.
+  All browsers on iOS are forced to use the WebKit,
+  and can't use their own browser engine.
+{{site.alert.end}}
 
-#### Requires preview JS-interop to access browser and JS APIs
+[currently experiencing a bug]: https://bugzilla.mozilla.org/show_bug.cgi?id=1788206
+[this bug]: https://bugs.webkit.org/show_bug.cgi?id=247394
 
-To support Wasm, Dart is shifting how it targets browser and JavaScript APIs.
-This shift prevents Dart code that uses `dart:html` or `package:js` from
-compiling to Wasm.
-Most platform-specific packages, such as [`package:url_launcher`][],
-use these libraries.
-As a result, they are currently incompatible with Wasm support in Flutter.
+#### Requires JS-interop to access browser and JS APIs
```

Comment:
Consider giving this a shorter fragment to link to. Maybe something like `{#js-interop-requirements}`?